.travis.yml

```diff
@@ -1,51 +1,20 @@
 sudo: required
-dist: trusty
+
 language: c
-matrix:
-   include:
-     - addons: 
-          postgresql: 9.3
-       env:
-          - POSTGRESQL=9.3
-       install:
-           - sudo apt-get install postgresql-server-dev-9.3
-           - sudo make install
-     - addons: 
-          postgresql: 9.4
-       env:
-          - POSTGRESQL=9.4
-       install:
-           - sudo apt-get install postgresql-server-dev-9.4
-           - sudo make install
-     - addons: 
-          postgresql: 9.5
-       install:
-           - sudo apt-get install postgresql-server-dev-9.5
-           - sudo make install
-       env:
-          - POSTGRESQL=9.5
-     - addons: 
-          postgresql: 9.6
-       install:
-           - sudo apt-get install postgresql-server-dev-9.6
-           - sudo make install
-       env:
-          - POSTGRESQL=9.6
-     - # addons: postgresql: 10
-       install:
-           - sudo service postgresql stop
-           - wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O - | sudo apt-key add -
-           - sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" >> /etc/apt/sources.list.d/pgdg.list'
-           - sudo apt-get update
-           - sudo apt-get install postgresql postgresql-contrib postgresql-client
-           - sudo apt-get install postgresql-server-dev-10
-           - sudo sh -c 'echo "port=5432" >> /etc/postgresql/10/main/postgresql.conf'
-           - sudo pg_ctlcluster 10 main restart
-           - sudo -u postgres psql  -c "create user travis with superuser";
-           - sudo make install
-       env:
-          - POSTGRESQL=10
+
+services: docker
+
+install:
+    - sed -e 's/${PG_VERSION}/'${PG_VERSION}/g Dockerfile.template > Dockerfile
```

Comment:
You don't need sed and a separate .template: you can define build-time variables in Dockerfile (with `ARG` directive and --`build-args` parameter to `docker build`).

Reply:
done